alacritty/src/display.rs

```diff
@@ -286,6 +287,21 @@ impl Display {
         size_info.cell_height = cell_height;
     }
 
+    /// Evict and replace glyph cache
+    pub fn handle_glyph_cache_refresh(&mut self, config: &Config) {
```

Comment:
Having a `handle_glyph_cache_refresh` and an `update_glyph_cache` function is a bit confusing, since the names do not clearly separate these two functions. We should make this more clear, renaming the old function is perfectly fine too.